coralproject/talk
# Fix deselect bug of action menu in ff

## What does this PR do?
- https://www.pivotaltracker.com/story/show/154183787

## How do I test this PR?
- In firefox, click on `actions` menu for banning / suspending a user
- Click outside and the menu should "deselect"
